data/foods/pizzaToppings.json

```diff
@@ -4,11 +4,16 @@
     "anchovies",
     "artichoke",
     "bacon",
+    "breakfast bacon",
+    "canadian bacon",
```

Comment:
That's a good idea. @webmaven, can you capitalize "Canadian" and then I'll
merge?